org.eclipse.winery.crawler/src/main/java/org/eclipse/winery/crawler/chefcookbooks/constants/Defaults.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+package org.eclipse.winery.crawler.chefcookbooks.constants;
+
+public class Defaults {
+
+    public static final String COOKBOOK_PATH = "C:/Users/Markus/Desktop/Work/cookbooks";
```

Comment:
Could you maybe make this configurable or use a generic path like `System.getProperty("user.home")`?